virttest/iscsi.py

```diff
@@ -822,6 +824,9 @@ class Iscsi(object):
     """
     @staticmethod
     def create_iSCSI(params, root_dir=data_dir.get_tmp_dir()):
+        cmd = "targetcli clearconfig confirm=true"
+        if process.system(cmd, shell=True) != 0:
+            self.skip("targetcli configuration unable to clear")
```

Comment:
`TestError` have to be raised instead of skip, also this step have to be moved down after the check for `targetcli` command availability.